cpu/atmega_common/periph/gpio.c

```diff
@@ -254,6 +428,77 @@ static inline void irq_handler(uint8_t int_num)
     __exit_isr();
 }
 
+#ifdef PCINT_NUM_BANKS
+/* inline function that is used by the PCINT ISR */
+static inline void pcint_handler(uint8_t bank, volatile uint8_t *mask_reg)
```

Comment:
instead of `volatile uint8_t *mask_reg`, why not directly pass the value as `uint8_t enabled_pcints` in this function?

Reply:
True. Good point.